src/rest_vol.c

```diff
@@ -3487,23 +3486,23 @@ RV_parse_server_version(char *HTTP_response, const void *callback_data_in, void
         FUNC_GOTO_ERROR(H5E_OBJECT, H5E_BADVALUE, FAIL, "server version was NULL");
 
     /* Parse server version into struct */
-    if (NULL == (version_field = strtok_r(version_response, ".", &saveptr)))
+    if (NULL == (version_field = strtok(version_response, ".")))
```

Comment:
I'd need to consult the YAJL docs, but it's possible this isn't safe since strtok modifies its input. MSVC is also generally POSIX-compliant for newer versions and should have either `strtok_r` or `strtok_s` available (see H5win32defs.h in HDF5).